partner_duplicate_mgmt/README.rst

```diff
@@ -53,6 +53,22 @@ A new list view shows alls duplicates. The user can:
 * exclude the duplicate by choice 
 
 
+Company and contact merger
+--------------------------
+Not possible
+
+
+Companies merger
+----------------
+Possible
```

Comment:
Please make complete sentences. You should only have one section explaining that merger between a company and a contact is not possible.